src/js/modules/network/requests/likes_request.js

```diff
@@ -0,0 +1,23 @@
+import NetWorkRequest from '../network_request';
+
+export default class LikesRequest {
+    #networRequest;
+
+    #path;
+
+    constructor(gammeID = '9JcQwe8YeiV9ciPMfpK9') {
```

Comment:
Again, avoiding putting the secret keys in your public repositories is highly recommended. Putting such a variable into a `.env` file could be significant for security.

I'd recommend the [Dotenv](https://www.npmjs.com/package/dotenv) package, but you're free to use whatever you want.